### DISCUSSION TITLE
Is there any option to reach target api through a proxy?

### DISCUSSION THREAD

**Discussion by koustabhchatt** [fortio/fortio#539] [Q&A] [answered]
Is there any option to reach target api through a proxy? if yes how to set it up in fortio command ?

**Comment by ldemailly** [comment]
I'm not sure what you're asking exactly?

you can set HTTPS_PROXY and HTTP_PROXY environment variables that golang supports for the `-stdclient`

**Reply by koustabhchatt** [reply]
I was looking for a option to put proxy details in fortio command / metadata itself like we can do with curl command.

**Reply by ldemailly** [reply]
That doesn't exist yet no for fortio (what's the flag name in curl?) - can be added if useful

**Reply by koustabhchatt** [reply]
From man curl:

-x, --proxy <[protocol://][user:password@]proxyhost[:port]>

     Use the specified HTTP proxy. 
     If the port number is not specified, it is assumed at port 1080.

**Comment by ldemailly** [comment, marked as answer]
Created #545 for the feature request
